lesson_07/objects/objects_app/src/main/java/com/codedifferently/lesson7/richhawkins/House.java

```diff
@@ -0,0 +1,61 @@
+package com.codedifferently.lesson7.richhawkins;
+
+import java.time.Year;
+
+public class House {
+  //     The class must have at least 5 member variables of at least 3 different types.
+  String streetName;
```

Comment:
your member variables should be `private`.

Reply:
member variables set to private. Created getters and setter so the variables are still accessible.